docs/source/1.0/spec/aws/aws-core.rst

```diff
@@ -123,7 +123,7 @@ generating client names in SDKs and for linking between services.
   the first character cannot be a number, and when using spaces, each space must be
   between two alphanumeric characters.
 * The value MUST NOT contain "AWS", "Aws", or "Amazon".
-* The value must not case-insensitively end with "API", "Client", or "Service".
+* The value SHOULD not case-insensitively end with "API", "Client", or "Service". **Note**: serveral services exist which end with ``Service`` or ``API`` for backwards compatibility reasons.
```

Comment:
Since this is called out later, we can probably just say:

```suggestion
* The value SHOULD NOT case-insensitively end with "API", "Client", or "Service".
```